src/electron/platform/android/rule-information-provider.ts

```diff
@@ -82,6 +82,139 @@ export class RuleInformationProvider {
                 }),
                 this.getStandardResultStatus,
             ),
+            ClassNameCheck: new RuleInformation(
+                'ClassNameCheck',
+                `${this.ruleLinkBaseUrl}/unsupported-item-type/`,
+                'View objects must have valid UI class names.',
+                [link.WCAG_4_1_2],
+                () => ({
+                    howToFixSummary:
+                        "Provide a className value within the element's AccessibilityNodeInfo that: 1. Closely describes the element's function, 2. Matches a class that extends android.view.View, 3. Is provided with the Android SDK or support libraries, and 4. Is as far down Android's class hierarchy as possible.",
+                    richResolution: {
+                        labelType: 'check',
+                        contentId: 'android/atfa/ClassNameCheck',
+                    },
+                }),
+                this.getStandardResultStatus,
+            ),
+            ClickableSpanCheck: new RuleInformation(
+                'ClickableSpanCheck',
+                `${this.ruleLinkBaseUrl}/link-purpose-unclear/`,
+                "A TextView widget must not contain ClickableSpan objects that aren't URLSpans with absolute URLs.",
+                [link.WCAG_4_1_2],
```

Comment:
Should this be 2.4.4?

Reply:
I checked with @iamrafan, and he and @ferBonnin had discussed this. In the docs PR, it references both 2.4.4 and 4.1.2, so that's what I'll do, too.